LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001.html

```diff
@@ -113,7 +113,7 @@
   promise_test(async t => {
     let results = await trySendingPlainStringToTrustedTypeSink(
       ["script"],
-      "header(Content-Security-Policy,require-trusted-types-for 'script''script' 'script',True)",
+      "header(Content-Security-Policy,require-trusted-types-for 'script''script',True)",
```

Comment:
The existing test is wrong this corrects it. This change will be exported separately by an associated Gecko patch.